build.gradle.kts

```
@@ -0,0 +1,38 @@
+plugins {
+    java
+    id("com.github.johnrengelman.shadow") version "6.1.0"
```

7.0.0 is the latest version
```suggestion
    id("com.github.johnrengelman.shadow") version "7.0.0"
```